databases/week1/db_w1.sql

```diff
@@ -0,0 +1,55 @@
+-- 1 Find out how many tasks are in the task table
+select count(id)
+from task;
+
+-- 2 Find out how many tasks in the task table do not have a valid due date
+select count(id)
+from task
+where due_date IS null;
+
+
+-- 3 Find all the tasks that are marked as done (what should I do with null response?)
+select id
+from task
+where status_id = 3;
+
+-- 4 Find all the tasks that are not marked as done
+select id
+from task
+where status_id != 3;
+
+-- 5 Get all the tasks, sorted with the most recently created first
+select id, created
+from task
+order by created desc;
+
+-- 6 Get the single most recently created task
+select id, created
+from task
+order by created desc
+limit 6;
+
+
+-- 7 Get the title and due date of all tasks where the title or description contains database
+select title, due_date
+from task
+where title is null or due_date is null;
+
+
+-- 8 Get the title and status (as text) of all tasks
+select title, name
```

Comment:
Valid query, but it's good practice to use aliases to make sure the results are as clear as possible.
Also, when using `JOIN`s it's good practice to use `<tableName>.<columnName>` so you or other people visiting the code later know where your columns come from.
```suggestion
select task.title, status.name as status
```